### PR TITLE
fix(blame-nvim): adjust to the new rewrite

### PR DESCRIPTION
**Pull request by manuuurino**
## 📑 Description

The blame.nvim plugin has been rewritten and "broke" this spec. Updated to this state https://github.com/FabijanZulj/blame.nvim/tree/39f6123b78bde45917de071a07eba17b136d9f2c.

### CODE REVIEW

File: lua/astrocommunity/git/blame-nvim/init.lua [hunk 1]
```diff
@@ -1,6 +1,7 @@
 return {
   "FabijanZulj/blame.nvim",
-  event = "User AstroGitFile",
+  cmd = "BlameToggle",
+  config = function(_, opts) require("blame").setup(opts) end,
```
Comment (mehalter) [inline lua/astrocommunity/git/blame-nvim/init.lua:4]:
```suggestion
  opts = {}
```

No need to redefine the default `config` function

**Review by mehalter — CHANGES_REQUESTED** [review]
Small nit

File: lua/astrocommunity/git/blame-nvim/init.lua (hunk 1 above)
Comment (manuuurino) [reply lua/astrocommunity/git/blame-nvim/init.lua]: Thanks, teached me something new :)!

**Review by mehalter — APPROVED** [review]
LGTM Thanks!

**Pull request merged**